### PR TITLE
feat: #3790 add topic select to contact form

### LINKED ISSUE #3789: feat(contact): Add 'Organisation' field ( free text) to contact form
**Is your feature request related to a problem? Please describe.**
CBCH - missing - why do we need this

**Describe the solution you'd like**
 Add 'Organisation' field ( free text) to contact form

**Describe alternatives you've considered**
Not adding the field 

**Additional context**
security ? remove code execution , spam considerations

### LINKED ISSUE #3790: Add 'Topic' field to contact form
**Is your feature request related to a problem? Please describe.**
CBCH - missing ? - why do we want / need this ?

**Describe the solution you'd like**
Contact form with optional 'Topic' field, topic input should be of type select with the following options:

- 'Data / sample request'
- 'Collaboration request'
- 'Information request'
- 'Other'

**Describe alternatives you've considered**
Not adding the fields

**Additional context**
Get options from db or hard code into app ?

### LINKED ISSUE #3791: feat(catalogue): Add {Cohort-Acronym} and {Topic} to contact request email subject
**Is your feature request related to a problem? Please describe.**
CBCH - missing - why do we whant this / needs this ?

**Describe the solution you'd like**
Add {Cohort-Acronym} and {Topic} to contact request email subject

**Describe alternatives you've considered**
Not adding the field

**Additional context**

- what about the context ( for example; 'Contact result for {Cohort-Acronym} on {Topic}' )
- security (? , free text to email subject ..

### PR DESCRIPTION
**Pull request by connoratrug**
What are the main changes you did:
Closes #3790 
Closes #3791 
Closes #3789 

example: http://localhost:3000/catalogue_demo/ssr-catalogue/testNetwork1/cohorts/testCohort1

how to test:
- setup mail service to given contact ( see docs on email service for catalogue)
- make sure to change/select a cohort contact email to a address that is owned by you ( i.e. do not send 'test' email to 'real' cohorts 
- click the contact button
- fill out the form ( with and without filling in topic field
- check email for message topic should be part of subject and message ( if filled in )

todo:
- [ ] updated docs in case of new feature
- [ ] added/updated tests
- [ ] added/updated testplan to include a test for this fix, including ref to bug using # notation

### CODE REVIEW

**Comment by EleanorHyde-UMCG** [comment]
I see no topic field in the contact form in 3875 prev...? Just this:

image

File: apps/nuxt3-ssr/components/content/ContentBlockIntro.vue [hunk 1]
```diff
@@ -45,34 +38,53 @@ fetchSetting("contactRecipientsQuery").then((resp) => {
 
 let showContactInformation = ref(false);
 
-const fields = reactive([
-  {
+const fields = reactive({
+  senderName: {
```
Comment (chinook25) [inline apps/nuxt3-ssr/components/content/ContentBlockIntro.vue:42]: Records are so much better to deal with than arrays <3

File: apps/nuxt3-ssr/components/content/ContentBlockIntro.vue [hunk 2]
```diff
@@ -83,35 +95,43 @@ const notificationMessage = ref("");
...
 const submitForm = async () => {
-  const senderName = fields.find((field) => field.name === "senderName");
-  const senderEmail = fields.find((field) => field.name === "senderEmail");
-  const senderMessage = fields.find((field) => field.name === "senderMessage");
   // Validate form fields
-
-  if (!senderEmail.fieldValue) {
-    senderEmail.hasError = true;
-    senderEmail.message = "Please enter a valid email address";
+  if (!fields.senderEmail.fieldValue) {
```
Comment (chinook25) [inline apps/nuxt3-ssr/components/content/ContentBlockIntro.vue:99]: In the constants file of the components lib is a regex for validating email addresses, might be an idea to apply it here?

**Review by chinook25 — CHANGES_REQUESTED** [review]
Code looks good, but I don't know how to test it from your h2t:

- Where can I find the docs? It's not in the readme
- Where can I find the contact button? https://preview-emx2-pr-3875.dev.molgenis.org/catalogue-demo/ssr-catalogue/

**Review by chinook25 — APPROVED** [review]

**Pull request merged**